pkg/client/client.go

```diff
@@ -222,6 +224,36 @@ func (c *Client) GetProxy(name string) (*configv1.Proxy, error) {
 	return c.oscclient.ConfigV1().Proxies().Get(name, metav1.GetOptions{})
 }
 
+func (c *Client) GetAPIServerAuthenticationConfigmap() (*v1.ConfigMap, error) {
```

Comment:
These two methods are somewhat out of place in this package. Note that no other methods in this package hardcore an explicit namespace or resource name. The fact that these two do indicates that these are utility methods rather than some general purpose client method. Instead, this method should probably be `GetConfigmap(cm *v1.ConfigMap)` or something similar.

Reply:
agreed, factoring this into a more abstract method.